### PR TITLE
Add RFC-0001/Overview

### PR DESCRIPTION
**Pull request by CjS77**
RFC-0001 is a very high-level summary of the Tari network. It describes the two main components, base layer and digital assets network, in the broadest terms.

Please limit debate to the paradigmatic axis; i.e. you should only argue the choices made here (e.g. Mimblewimble vs Proof-of-waffles-based tangle) that are at ±the same conceptual level by commenting here, or adding to this RFC's issue once it has been merged.

Please refrain from starting debate that requires further detail (there are other RFCs for that).

### CODE REVIEW

File: RFC/src/RFC-0001_overview.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+### The Base Layer
+
+_Refer to [RFC-0100/BaseLayer](RFC-0100_BaseLayer.md) for more detail_.
+
+The Tari base layer has the following primary features:
+
+* Proof of work-based blockchain using Nakamoto consensus
+* Transactions and blocks based on the [Mimblewimble] protocol
+
+[Mimblewimble] is an exciting new blockchain protocol that offers some key advantages over other [UTXO]-based
```
Comment (philipr-za) [inline RFC/src/RFC-0001_overview.md]: The Mimblewimble link seems broken

File: RFC/src/RFC-0001_overview.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+Given Tari's relationship with Monero, a merge-mined strategy with Monero makes the most sense, but the PoW mechanism
+SHOULD be written in a way that makes it relatively easy to code, implement and switch to a different strategy in the
+future.
+
+### The Digital Assets Network
+
+A more detailed proposal for the digital assets network is presented in [RFC-0300/DAN](RFC-0300_DAN.md). Digital assets
+_are discussed in more detail in [RFC-0310/Assets](RFC-0310_Assets.md)._
+
+The Tari digital assets network (DAN) consists of a peer-to-peer network of [Validator nodes]. These nodes ensure the
```
Comment (philipr-za) [inline RFC/src/RFC-0001_overview.md]: Validator nodes link is broken

File: RFC/src/RFC-0001_overview.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+There are two major digital entities on the Tari network: The coins that are the unit of transfer for the Tari
+cryptocurrency, and the digital assets that could represent anything from tickets to in-game items.
+
+Tari coins are the fuel that drives the entire Tari ecosystem. But they share many of the properties of money, and so
+security is a non-negotiable requirement. In a cryptocurrency context, this is usually achieved by employing a
+decentralised network running a censorship-resistant protocol like Nakamoto consensus over a proof of work blockchain.
+As we know, proof of work blockchains are not scalable, or terribly fast.
+
+On the other hand, the Tari network will be used to create and manage digital assets.
```
[the remark this answers is not in the document]
Comment (CjS77) [reply RFC/src/RFC-0001_overview.md]: Yeah. Good catch

File: RFC/src/RFC-0001_overview.md (hunk 2 above)
Comment (CjS77) [reply RFC/src/RFC-0001_overview.md]: fixed

**Pull request merged**